cyhy_report/customer/report.mustache

```diff
@@ -825,6 +829,26 @@ Details on the below findings can be found in ``findings.csv" in
 	  \scalegraphicsnofig{max-age-active-kevs} \RCSourceSansProSemiBoldTen{MAX AGE OF ACTIVE KEVS}
 \end{minipage}}  % END: KEV RESPONSE TIME
 
+\newpage
+<<#owner_is_federal_executive>>
+\section{Binding Operational Directive 23-02 --- Mitigating the Risk from Internet-Exposed Management Interfaces}<</owner_is_federal_executive>><<^owner_is_federal_executive>>\section{Mitigating the Risk from Internet-Exposed Management Interfaces}<</owner_is_federal_executive>>
+\label{sec:internet-exposed-management-interfaces}
+
+Threat actors often use certain classes of network devices to gain unrestricted access to organizational networks leading to full scale compromises. Inadequate security, misconfigurations, and out-of-date software make these devices more vulnerable to exploitation. The risk is further compounded if device management interfaces are connected directly to, and accessible from, the public-facing Internet. Most device management interfaces are designed to be accessed from dedicated physical interfaces and/or management networks and are not meant to be accessible directly from the public Internet.
+
+\gls{CISA} <<#owner_is_federal_executive>>issued \href{https://www.cisa.gov/news-events/directives/binding-operational-directive-23-02}{Binding Operational Directive (BOD) 23-02} to push the federal government to take steps toward reducing the attack surface created by insecure or misconfigured management interfaces across certain classes of devices. The BOD requires networked management interfaces (NMIs) using certain protocols over the Internet to <</owner_is_federal_executive>><<^owner_is_federal_executive>>recommends that networked management interfaces (NMIs) using certain protocols over the Internet <</owner_is_federal_executive>>be removed from the public Internet or to be protected by capabilities that enforce access control to the interface through a policy enforcement point separate from the interface itself as part of a Zero Trust Architecture (ZTA) within 14 days of discovery.
```

Comment:
```suggestion
\gls{CISA} <<#owner_is_federal_executive>>issued \href{https://www.cisa.gov/news-events/directives/binding-operational-directive-23-02}{Binding Operational Directive (BOD) 23-02} to push the federal government to take steps toward reducing the attack surface created by insecure or misconfigured management interfaces across certain classes of devices. The BOD requires networked management interfaces (NMIs) using certain protocols over the Internet to <</owner_is_federal_executive>><<^owner_is_federal_executive>>recommends networked management interfaces (NMIs) using certain protocols over the Internet to <</owner_is_federal_executive>>be removed from the public Internet or to be protected by capabilities that enforce access control to the interface through a policy enforcement point separate from the interface itself as part of a Zero Trust Architecture (ZTA) within 14 days of discovery.
```

Reply:
As I was looking through your suggestion, I realized that my output differed slightly from the text provided in the [requirements](https://github.com/cisagov/cyhy-system/issues/89).  I made corrections in https://github.com/cisagov/cyhy-reports/pull/89/commits/16a1f76fd9135570fa29369ba180b534f5fc5ade.

If you still have an issue with this, then I think you might have a disagreement with the text provided in the requirements.

Reply:
That commit looks good to me.  Thank you!